core/lib/protobuf_config/src/proto/config/contracts.proto

```diff
@@ -17,6 +18,8 @@ message L1 {
   optional string multicall3_addr = 6; // required; H160
   optional string base_token_addr = 7; // required; H160
   optional string chain_admin_addr = 8; // required; H160
+  optional string base_token_asset_id = 9; // required; H256
+  optional string predeployed_l2_weth_token_address = 10; // optional; H160
```

Comment:
`predeployed_l2_weth_token_address` should be put to `message L2` if it's L2 address